src/settings.ts

```diff
@@ -112,8 +112,9 @@ export const SCHEDULE_CURRENT_TIMELINE_MS = 60 * 1000; // 1 min
 export const EVENT_STATUS_REFRESH_MS = 60 * 1000; // 1 min
 
 export const ROOM_IMAGE_WIDTH_PX = 300;
-export const MAX_IMAGE_FILE_SIZE_BYTES = 1024 * 2000;
-export const MAX_IMAGE_FILE_SIZE_TEXT = "2MB";
+export const MAX_IMAGE_FILE_SIZE_MB = 2;
+export const MAX_IMAGE_FILE_SIZE_BYTES = MAX_IMAGE_FILE_SIZE_MB * 1024 * 1024;
+export const MAX_IMAGE_FILE_SIZE_TEXT = `${MAX_IMAGE_FILE_SIZE_MB}MB`;
```

Comment:
The name should imply that this is in MB.

Reply:
Maybe `MAX_IMAGE_FILE_SIZE_MB_TEXT`

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1988/commits/1a9dbe540ef6febdd35859a34957baab44a7e559